src/scripts/updateAnswers.js

```diff
@@ -0,0 +1,50 @@
+const { v4: uuidv4 } = require('uuid');
```

Comment:
Add a summary of code comment at the start of the script to explain what it's for and how to use it.